test/Microsoft.AspNet.Razor.Runtime.Test/DefaultTagHelperContentTest.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    public class DefaultTagHelperContentTest
+    {
+        [Fact]
+        public void CanSetContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World!";
+
+            // Act
+            tagHelperContent.SetContent(expected);
+
+            // Assert
+            Assert.Equal(expected, tagHelperContent.ToString());
```

Comment:
use `GetContent()` to show correct non-debugging use of `TagHelperContent`.  only exception should be a specific test or two of the `ToString()`.